apisix/plugins/tcp-logger.lua

```diff
@@ -106,7 +108,8 @@ local function remove_stale_objects(premature)
 
     for key, batch in ipairs(buffers) do
         if #batch.entry_buffer.entries == 0 and #batch.batch_to_process == 0 then
-            core.log.debug("removing batch processor stale object, route id:", tostring(key))
+            core.log.warn("removing batch processor stale object, conf: ",
+                          core.json.delay_encode(key))
```

Comment:
Should we also update? https://github.com/apache/apisix/blob/a52b5e9b3b00da40705144c868cfe3a67de47b8d/apisix/plugins/http-logger.lua#L183

Reply:
Yes.

Reply:
@spacewander Fixed, also fixed it in kakfa-logger.lua